### PR TITLE
Task#68079: FE Chi tiet mot lich hen

### PR DESCRIPTION
**Pull request by dangvm-0369**
## Related Tickets:
- [Task#68079](https://edu-redmine.sun-asterisk.vn/issues/68079)

## WHAT this PR does?
- FE Xem chi tiet mot lich hen

## Notes Impacted Areas:
- N/A.

## Attachments:
ShowAOrder

### CODE REVIEW

**Comment by dangvm-0369** [comment]
ready

File: resources/js/Pages/orders/Show.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import Authenticated from '@/Layouts/Authenticated';
+import { Head } from '@inertiajs/inertia-react';
+import Button from '@/Components/Button';
+import {Descriptions, Table, Tag, PageHeader} from 'antd';
+import { useLang } from '../../Context/LangContext';
```
Comment (ghost) [inline resources/js/Pages/orders/Show.jsx]:
```suggestion
import { Descriptions, Table, Tag, PageHeader } from 'antd';
import { useLang } from '../../Context/LangContext';
```

File: resources/js/Pages/orders/Index.jsx [hunk 2]
```diff
@@ -49,7 +49,9 @@ export default function Index(props) {
             render: (text, record) => {
                 return (
                     <div>
-                        <Button className="hover:bg-slate-300 hover:text-gray-950">{lang.get('strings.Detail')}</Button>
+                        <a href={route('orders.show', {order : record.id})}>
```
Comment (ghost) [inline resources/js/Pages/orders/Index.jsx]:
```suggestion
                        <a href={ route('orders.show', { order : record.id }) }>
```

**Comment by dangvm-0369** [comment]
ready

**Pull request merged**